sdk/Lusid.Sdk/Utilities/ApiExceptionExtensions.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Lusid.Sdk.Utilities
 {
+    /// <summary>
+    /// Utility functions
```

Comment:
Extension to deserialise LUSID exceptions contained within the ApiException.ErrorContent